src/sentry/templatetags/sentry_assets.py

```diff
@@ -35,3 +35,17 @@ def crossorigin():
         # They share the same domain prefix, so we don't need CORS
         return ''
     return ' crossorigin="anonymous"'
+
+
+@register.simple_tag
+def locale_js_include(request):
```

Comment:
You should also be able to do:  `takes_context=True` as a property to the `simple_tag`.

Then you don't need to explicitly pass in `request`.

So:

```python
@register.simple_tag(takes_context=True)
def locale_js_include(context):
    request = context['request']
    ....
```

Then your template tag is just `{% locale_js_include %}`